doc/guides/modules.rst

```diff
@@ -487,6 +505,55 @@ variables. For example:
 
   include(${ZEPHYR_CURRENT_MODULE_DIR}/cmake/code.cmake)
 
+.. _modules_module_root:
+
+Module root
+===========
+
+Zephyr repository contains build files certain known Zephyr modules.
+
+You can add additional paths with build files for Zephyr module by using the
+``-DMODULE_ROOT`` parameter to the CMake build system. This will tell the build
+system to also search this particular path for Zephyr modules using
```

Comment:
Can you clarify whether it is possible to add multiple additional module roots?

Reply:
It is.
Will highlight that when updating the description.

Reply:
done